CMakeLists.txt

```diff
@@ -361,6 +361,24 @@ include_directories("${PROJECT_SOURCE_DIR}/src")
 find_package(LibUV REQUIRED)
 include_directories(SYSTEM ${LIBUV_INCLUDE_DIRS})
 
+if(WIN32)
+  list(APPEND CMAKE_REQUIRED_INCLUDES "${LIBUV_INCLUDE_DIRS}")
+  list(APPEND CMAKE_REQUIRED_LIBRARIES "${LIBUV_LIBRARIES}")
```

Comment:
Are these two lines necessary after we did `find_package(LibUV REQUIRED)` above?

Reply:
Without these two lines, `check_c_source_compiles` below will always fail.